canonical-web-and-design/ubuntu.com
# Create vm install instructions

## Done

- Create vm install instructions

**NOTE** - the doc is not completed, so just getting something in place for now... missing ports in appliance.yaml for new appliances

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex/vm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1G8IJoHrhd1RP3dHqyiasteMxPrxCwW5Sf0Vh3AY2SHQ/edit#heading=h.4p9fkvn8lrpu) and resolve comments
